qemu/tests/cfg/balloon_small_size.cfg

```diff
@@ -0,0 +1,12 @@
+- balloon_small_size:
+    virt_test_type = qemu
+    no Win2000
+    only Windows
```

Comment:
In fact, Linux can't handle the "small_size" at present. But developer said they will realize it in the future. So, for present test case, i use "only Windows".